drivers/mfrc522/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_MFRC522
```

Comment:
You probably need to `rsource` this file from `drivers/Kconfig`

Reply:
Done